README.md

```diff
@@ -1 +1,31 @@
-# animal-shelter
\ No newline at end of file
+# Animal shelter
+
+## Motivation
+
+The idea of this project is to help with understanding and consolidate knowledge about different technologies used in programming. 
+
+## About project
+
+### Aim
+The project was created to learn and practice using these skills in a real platform with real problems. 
+
+The `Animal shelter` is an application that focuses on simple operations in the beginning and in process of time will be developed and new functionalities will be added. 
```

Comment:
This sentence is too long. Could you please create two of this one? Maybe "... that initially focuses on simple operations. With time new functionalities will be added."

Reply:
Done